lib/inputs/index.js

```diff
@@ -63,12 +57,7 @@ class Inputs {
  */
 module.exports.BigqueryInputs = class BigqueryInputs extends Inputs {
   constructor(records) {
-    super(
-      records,
-      new DovetailDownloads(records),
-      new DovetailImpressions(records),
-      new PixelTrackers(records),
```

Comment:
How pixel trackers different than pingbacks?

Reply:
We were actually abusing this repo a bit to get [our own pixel-tracker](https://github.com/PRX/pixel.prx.org) data into BigQuery.  Since a lot of the fields are the same as a podcast-download.  In fact, you can still see that in the BQ `pixels.pridotorg` dataset.  But we used that pixel tracker for plain website analytics, not podcast pingbacks.